docs/ja/front/component/ma-pro-table.md

```
@@ -109,22 +149,22 @@ addPlugin({
 ![テーブルツールバー](https://s21.ax1x.com/2024/11/03/pArlfRU.png)
 
 ::: tip 説明
-`ma-pro-table` には[スロット](#slot)でここを拡張することもできます。一時的な機能にはスロットを使用し、システム全体で必要な場合は `API` で拡張することをお勧めします。
+`ma-pro-table`には[スロット](#slot)もあり、ここを拡張できます。一度限りの機能にはスロットを使用し、システム全体で必要な場合は`api`で拡張することをお勧めします。
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)